packages/tokens/paragraph-tokens/tokens.json

```diff
@@ -50,6 +50,20 @@
             "nl.nldesignsystem.figma-implementation": true
           },
           "$type": "lineHeights"
+        },
+        "margin-block-end": {
+          "$extensions": {
+            "nl.nldesignsystem.css-property-syntax": "<length>",
+            "nl.nldesignsystem.figma-implementation": false
```

Comment:
@jeffreylauwers ik heb dit op `false` gezet omdat Figma geen margin kent maar ik weet niet of je hier misschien wat anders slims voor hebt.

Reply:
Denk dat het goed is zo.